vizro-core/tests/unit/vizro/models/_components/form/test_slider.py

```diff
@@ -63,10 +63,10 @@ def test_create_slider_mandatory(self):
 
         assert hasattr(slider, "id")
         assert slider.type == "slider"
+        assert slider.marks is None
         assert slider.min is None
         assert slider.max is None
         assert slider.step is None
-        assert slider.marks is None
```

Comment:
Can you add unit tests if these are not in here yet:
- marks == {} if step is set
- marks is None if step is not set
- marks is None if step is set and marks=None